api/src/main/resources/ahc-default.properties

```diff
@@ -0,0 +1,90 @@
+play.shaded.ahc.org.asynchttpclient.threadPoolName=AsyncHttpClient
```

Comment:
there is a bug in play where it cannot read properties from the shaded jar - so I had to fix that by copy pasting default properties inside this project